### PR TITLE
Add support for fuel-core 0.11

### PR DESCRIPTION
**Pull request by digorithm**

### CODE REVIEW

**Review by Salka1988 — DISMISSED** [review]

File: packages/fuels-test-helpers/src/node.rs [hunk 1]
```diff
@@ -42,7 +41,7 @@ impl Config {
             predicates: false,
             manual_blocks_enabled: false,
             vm_backtrace: false,
-            silent: true,
+            silent: false, // Temp for debugging.
```
Comment (mohammadfawaz) [inline packages/fuels-test-helpers/src/node.rs:44]: Is this intentional?

**Review by mohammadfawaz — COMMENTED** [review]
Looks good! Just one small comment.

**Review by iqdecay — APPROVED** [review]
:ship:, except for the question of Mohammad. Neat :)

File: packages/fuels-test-helpers/src/node.rs (hunk 1 above)
Comment (digorithm) [reply packages/fuels-test-helpers/src/node.rs:44]: Nope, that was for debugging the fuel-core binary behaviour inside the CI :-)

**Review by mohammadfawaz — APPROVED** [review]
:shipit:

**Pull request merged**